testsuite/cluster-test/src/experiments/load_test.rs

```diff
@@ -101,28 +126,24 @@ impl Experiment for LoadTest {
 
         if self.mempool {
             // spawn mempool load test
-            let (mempool_sender, mempool_events) = stubbed_node
-                .mempool_handle
-                .take()
-                .expect("missing mempool network handles");
-            mempool_task = Some(tokio::task::spawn(mempool_load_test(
-                duration,
-                mempool_sender,
-                mempool_events,
-            )));
+            for (mempool_sender, mempool_events) in mempool_handlers {
+                mempool_task.push(Some(tokio::task::spawn(mempool_load_test(
+                    duration,
+                    mempool_sender,
+                    mempool_events,
+                ))));
+            }
         }
 
         if self.state_sync {
             // spawn state sync load test
-            let (state_sync_sender, state_sync_events) = stubbed_node
-                .state_sync_handle
-                .take()
-                .expect("missing state sync network handles");
-            state_sync_task = Some(tokio::task::spawn(state_sync_load_test(
-                duration,
-                state_sync_sender,
-                state_sync_events,
-            )));
+            for (state_sync_sender, state_sync_events) in state_sync_handlers {
+                state_sync_task.push(Some(tokio::task::spawn(state_sync_load_test(
```

Comment:
why this is a Some(..)?

Reply:
hmm, I refactor base on old logic. u are right, after we have vec we do not need some any more.